### PR TITLE
remove ropsten and kovan

### PR DESCRIPTION
**Pull request by azf20**

### CODE REVIEW

File: pages/ar/developing/developer-faqs.mdx [hunk 1]
```diff
@@ -110,7 +110,7 @@ curl -X POST -d '{ "query": "{indexingStatusForCurrentVersion(subgraphName: \"or
 - Ethereum mainnet
 - Kovan
 - Rinkeby
-- Ropsten
+- Goerli
```
Comment (benface) [inline pages/ar/developing/developer-faqs.mdx:113]: Goerli was already in that list (just below).

Comment (benface) [reply pages/ar/developing/developer-faqs.mdx:113]: Also, there is an instance of "Kovan" at the bottom of this list.

**Review by benface — COMMENTED** [review]
Should the title of this PR be "remove ropsten and kovan"?

File: pages/ar/developing/developer-faqs.mdx (hunk 1 above)
Comment (azf20) [reply pages/ar/developing/developer-faqs.mdx:112]: Good catch, hadn't saved that file

**Review by benface — APPROVED** [review]

**Review by Data-Nexus — COMMENTED** [review]
These changes all look good to me.

**Pull request merged**